### PR TITLE
Added changelog for 3.21.6

### PR DESCRIPTION
**Pull request by craigcomstock**

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thanks for submitting a pull request! Maybe @larsewi can review this?

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,38 @@
+3.21.6:
+	- Added logging CFEngine component related SELinux denials in cf-support
+	  (ENT-12137)
+	- Also ignore IPv6 interface info
+	  Agent now also ignores interfaces listed in ignore_interfaces.rx when
```
Comment (nickanderson) [inline ChangeLog:5]:
```suggestion
	- Agent now also ignores interfaces listed in ignore_interfaces.rx when
```

File: ChangeLog [hunk 2]
```diff
@@ -1,3 +1,38 @@
...
+	  looking for IPv6 interface info. Variables such as
+	  'default:sys.hardware_mac[<INTERFACE>]' will no longer be defined for
+	  ignored interfaces.
+	  Ticket: (ENT-11840)
```
Comment (nickanderson) [inline ChangeLog:9]:
```suggestion
	  ignored interfaces. (ENT-11840)
```

File: ChangeLog [hunk 3]
```diff
@@ -1,3 +1,38 @@
...
+	    permission during remote copy. Previously it would be created with
+	    default permissions.
+	  - The destination file (i.e., '<FILENAME>') is no longer deleted on
+	    backup during file copy.  Previously it would be renamed to
+	    '<FILENAME>.cfsaved', causing the original file to dissappear. Now an
+	    actual copy of the original file with the same permissions is created
+	    instead.
+	  As a result, there will no longer be a brief moment where the original
+	  file is inaccessible.
+	  Ticket: (ENT-11988)
```
Comment (nickanderson) [inline ChangeLog:22]:
```suggestion
	  file is inaccessible. (ENT-11988)
```

File: ChangeLog [hunk 4]
```diff
@@ -1,3 +1,38 @@
...
+	- Fixed package promises with only promisers and no other attributes
+	  (CFE-4315, CFE-4398, CFE-4408)
+	- Fixed packagesmatching() function segfaulting on Windows platform
+	  (ENT-11900)
+	- Trailing newline on insert_tree promisers no longer removes ending tag for select_xpath
+	  (CFE-3806)
+	- commands promises with exit codes not matching any
+	             _returncodes attributes from classes body now log and
+	             error message not just an info message
+	  (CFE-4429, ENT-12103)
```
Comment (nickanderson) [inline ChangeLog:34]:
```suggestion
	- commands promises with exit codes not matching any
	   _returncodes attributes from classes body now log and
	   error message not just an info message (CFE-4429, ENT-12103)
```

**Review by nickanderson — APPROVED** [review]
I made some minor cleanup suggestions

File: ChangeLog [hunk 5]
```diff
@@ -1,3 +1,38 @@
...
+	  'default:sys.hardware_mac[<INTERFACE>]' will no longer be defined for
+	  ignored interfaces.
+	  Ticket: (ENT-11840)
+	- Atomic copy_from in files promise
+	  Changes to 'files' promise in 'copy_from' attribute:
+	  - The new file (i.e., '<FILENAME>.cfnew') is now created with correct
+	    permission during remote copy. Previously it would be created with
+	    default permissions.
+	  - The destination file (i.e., '<FILENAME>') is no longer deleted on
+	    backup during file copy.  Previously it would be renamed to
```
Comment (larsewi) [inline ChangeLog:16]:
```suggestion
	    backup during file copy. Previously it would be renamed to
```

**Review by larsewi — APPROVED** [review]

**Pull request merged**